BookingsApi/BookingsApi.Infrastructure.Services/AsynchronousProcesses/NewJudiciaryAddedAsynchronousProcesses.cs

```diff
@@ -24,8 +24,8 @@ public async Task Start(VideoHearing videoHearing, IList<JudiciaryParticipant> n
             await ((IPublishJudiciaryParticipantsEvent)_publisherFactory.Get(EventType.JudiciaryParticipantAddedEvent)).PublishAsync(videoHearing, newJudiciaryParticipants);
             if (sendNotification)
             {
-                await _publisherFactory.Get(EventType.HearingNotificationForJudiciaryParticipantEvent)
-                    .PublishAsync(videoHearing);
+                await ((IPublishJudiciaryParticipantsEvent)_publisherFactory.Get(EventType.HearingNotificationForJudiciaryParticipantEvent))
```

Comment:
please could you add an API integration test for this scenario?

Reply:
test added